src/components/footer/www/footer.jsx

```diff
@@ -20,34 +20,34 @@ var Footer = React.createClass({
                     <div className="lists">
                         <dl>
                             <dd>
-                                <a href="/about/">
+                                <a href="/about">
                                     <FormattedMessage id='general.aboutScratch' />
                                 </a>
                             </dd>
                             <dd>
-                                <a href="/jobs/">
+                                <a href="/jobs">
                                     <FormattedMessage id='general.jobs' />
                                 </a>
                             </dd>
                             <dd>
-                                <a href="/contact-us/">
+                                <a href="/contact-us">
```

Comment:
This still points to a scratchr2 page – I think we want to leave the trailing slash in for this one.